cypress/integration/homework/classes/HomePage.js

```diff
@@ -0,0 +1,70 @@
+class HomePage {
+
+
+    clearButton = "#clearButton";
+    actionDropDown = "#selectOperationDropdown";
+    calculateButton = "#calculateButton";
+    numberField1 = "#number1Field";
+    numberField2 = "#number2Field";
+    integersOnlyCheckbox = "#integerSelect";
+
+
+    checkIfFieldsEnabled(field) {
+        cy.get(field).should('be.enabled')
+    }
+
+    enterFieldsWith(number1, number2, action) {
+        cy.get(this.numberField1).clear()
+        cy.get(this.numberField1).type(number1)
+        cy.get(this.numberField2).clear()
+        cy.get(this.numberField2).type(number2)
+        cy.get(this.actionDropDown).select(action)
+        cy.get(this.calculateButton).click()
+    }
+    checkAnswer(answer) {
+        cy.get('input[name="numberAnswer"]').invoke('val').should('eq', answer)
+    }
+    checkTestClearButton() {
+        this.checkIfFieldsEnabled(this.clearButton)
+        cy.get(this.clearButton).click()
+        cy.get('input[name="numberAnswer"]').invoke('val').should('eq', '')
+    }
+    TestIntegersOnlyCheckbox(action) {
+        var number = '0.0'
```

Comment:
I think this method is too huge. Also, you are using different numbers than in your tests. Is this really needed? Maybe it is possible to check some values in main tests and this method used only for checking/unchecking the checkbox?